css/_responsive.scss

```diff
@@ -166,6 +166,19 @@
     }
 }
 
+@media (min-width: 100px) and (max-width: 320px) {
+    .smiley-input {
+        display: none;
+    }
+    .shift-right .focus-lock > div > div {
```

Comment:
This selector looks very fragile. Can't we add a class or something to the element?